src/main/java/com/questhelper/requirements/item/ItemRequirement.java

```diff
@@ -607,7 +611,11 @@ private ItemAndLastUpdated[] containersToCheckDefault()
 		containers.add(QuestContainerManager.getEquippedData());
 
 		if (!equip) containers.add(QuestContainerManager.getInventoryData());
-		if (shouldCheckBank) containers.add(QuestContainerManager.getBankData());
+		if (shouldCheckBank)
```

Comment:
imo the name of `containersToCheckDefault` is a bit misleading, I would personally move the logic of this function to the `check` function